hack/update-codegen.sh

```diff
@@ -60,25 +60,20 @@ GV_DIRS_CSV=$(IFS=',';echo "${GV_DIRS[*]// /,}";IFS=$)
 
 # This can be called with one flag, --verify-only, so it works for both the
 # update- and verify- scripts.
-${clientgen} --output-base "${KUBE_ROOT}/vendor" --output-package="k8s.io/client-go" --clientset-name="kubernetes" --input-base="k8s.io/kubernetes/vendor/k8s.io/api" --input="${GV_DIRS_CSV}" --go-header-file ${KUBE_ROOT}/hack/boilerplate/boilerplate.generatego.txt "$@"
+${clientgen} --output-base "${KUBE_ROOT}/vendor" --output-package="k8s.io/client-go" --clientset-name="kubernetes" --input-base="k8s.io/kubernetes/vendor/k8s.io/api" --input="${GV_DIRS_CSV}" --go-header-file "${KUBE_ROOT}/hack/boilerplate/boilerplate.generatego.txt" "$@"
 
-listergen_external_apis=(
-$(
-  cd ${KUBE_ROOT}/staging/src
-  find k8s.io/api -name types.go | xargs -n1 dirname | sort
-)
+mapfile -t listergen_external_apis < <(
```

Comment:
change reason:
```
In ./hack/update-codegen.sh line 66:
$(
^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```
Ref: https://github.com/koalaman/shellcheck/wiki/SC2207